dev_guide/deployments.adoc

```diff
@@ -396,14 +419,34 @@ no strategy is specified on a deployment configuration.
 [[rolling-strategy]]
 === Rolling Strategy
 
-The rolling strategy performs a rolling update and supports
-xref:lifecycle-hooks[lifecycle hooks] for injecting code into the deployment
-process.
+A rolling deployment slowly replaces instances of the previous version of an application
+with instances of the new version of the application. A rolling deployment typically waits
+for new pods to become *ready* via a *readiness check* before scaling down the old components.
+If a significant issue occurs, the rolling deployment can be aborted.
```

Comment:
> describe what is the significant issue (failed readiness probe? quota? etc)

I can only give a few examples but eventually I don't think we need to do it here (can be part of the section dedicated to debugging).